packages/generator/src/WireOntologyDefinition.ts

```diff
@@ -23,4 +23,5 @@ export interface WireOntologyDefinition {
   objectTypes: { [key: string]: gateway.components.ObjectTypeV2 };
   actionTypes: gateway.components.ActionTypeV2[];
   queryTypes: gateway.components.QueryTypeV2[];
+  linkTypes: Record<string, gateway.components.LinkTypeSideV2[]>;
```

Comment:
I'm not committed to this format at all. I just wanted to get something in here.